action.yaml

```diff
@@ -42,6 +42,9 @@ runs:
     - name: Remove unwanted packages
       run: sudo apt-get purge -y python3-simplejson python3-pyasn1-modules postgresql* || true
       shell: bash
+    - name: Ensure upgrade
```

Comment:
don't we want to run it earlier? like as a first step?

Reply:
ack